store/tikv/gcworker/gc_worker.go

```diff
@@ -1038,11 +1050,6 @@ func (w *GCWorker) genNextGCTask(bo *tikv.Backoffer, safePoint uint64, key kv.Ke
 func (w *GCWorker) doGC(ctx context.Context, safePoint uint64, concurrency int) error {
 	metrics.GCWorkerCounter.WithLabelValues("do_gc").Inc()
 
-	err := w.saveSafePoint(w.store.GetSafePointKV(), tikv.GcSavedSafePoint, safePoint)
```

Comment:
There are other functions that call `doGC`, see `RunGCJob`, if you simply remove `saveSafePoint` here, those function will be affected

Reply:
but RunGCJob is useless now, @MyonKeminta PTAL

Reply:
The API is not removed, you can't keep others from calling it. @winkyao

Reply:
already added save safe point time here.

Reply:
@winkyao It is not useless. Comments state that _It is exported for kv api_.

Reply:
Yes, the RunGCJob and RunDistributedGCJob is useful for kv only clusters.